### ISSUE TITLE
Info on the ECD WG Initiatives web page needs to be updated

### ISSUE THREAD

**Issue by jpersonna** [EclipseFdn/ecdtools.eclipse.org#37]
There have been some important updates to the slides for the ECD WG Initiatives. The updates were done prior to the Community Meeting that was held in May. It would be great if the ECD WG Initiatives web page (https://ecdtools.eclipse.org/initiatives/) could be updated to reflect those changes.

Thank you!

**Comment by brianking** [comment]
@jpersonna Could you summarize here what those changes are please.

**Comment by brianking** [comment]
@jpersonna This has been updated, please check to see if it meets your expectations based on why you opened the issue. Thanks.

Closing for now.
